rcl/test/rcl/test_graph.cpp

```diff
@@ -63,6 +63,9 @@ class CLASSNAME (TestGraphFixture, RMW_IMPLEMENTATION) : public ::testing::Test
   rcl_wait_set_t * wait_set_ptr;
   const char * test_graph_node_name = "test_graph_node";
 
+  rmw_topic_info_array_t * topic_info_array;
+  const char * topic_name = "valid_topic_name";
```

Comment:
nit: 
you probably want:
```c
const char * const topic_name = "valid_topic_name";
```
since that's a "constant pointer to constant chars"